src/css/exchange/exchangeDiagnostic.css

```diff
@@ -73,3 +73,7 @@
 .image-container:hover .overlay-transition{
   opacity: 1;
 }
+
+.select-domain .oui-field__component {
```

Comment:
Please avoid creating new CSS rules. If you have an issue with the style of ui-kit, you can talk about it with @AxelPeter :)

Reply:
I have asked Axel on this. OUI_Select adds margin-bottom by default. There is no way to remove it using input configuration. May be component needs to be modified to support this.

Reply:
Axel confirmed that this is the only way to do it as of now.